research-software-directory/RSD-as-a-service
# 301 featured mentions

# Featured mentions for software and projects

Closes #301
Closes #295 

Changes proposed in this pull request:
* Creating new section Highlights in software edit mentions, project impact and output
* User can manually create Highlight mention where he/she can provide image_url. These items are pinned to top of the mention section 
* All mention types are offered in manual mode

How to test:
* `docker-compose down --volumes`
* `docker-compose build` to build
* `docker-compose up` to start
*  login as professor1
*  create software,
* add mention using add button, select Highlight mention type, provide all info, use info [from this page as example](https://www.sciencedirect.com/book/9780124166325/handbook-of-statistical-analysis-and-data-mining-applications) for url and image url
* confirm that Highlight section is shown in edit module and the item is pinned
* repeat this for project impact and output sections


## Example software

![RSD-mentions-20220604](https://user-images.githubusercontent.com/9204081/172055326-f5b007ab-2f43-4ada-83f9-559e48382d5c.png)

## Example project input

![image](https://user-images.githubusercontent.com/9204081/172387730-82643813-7a72-49ae-b56f-2b22d6409b34.png)

PR Checklist:
